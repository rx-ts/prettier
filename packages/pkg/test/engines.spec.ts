import prettier from 'prettier'

test('engines', () => {
  const fixture = {
    engines: {
      npm: 'joker',
      node: 'batman',
    },
  }

  const input = JSON.stringify(fixture, null, 2)
  const output = prettier.format(input, {
    filepath: 'package.json',
    parser: 'json-stringify',
    plugins: ['prettier-plugin-pkg'],
  })

  expect(output).toMatchSnapshot()
})
