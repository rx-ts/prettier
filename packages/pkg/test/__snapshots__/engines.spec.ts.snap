// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`engines 1`] = `
"{
  \\"engines\\": {
    \\"node\\": \\"batman\\",
    \\"npm\\": \\"joker\\"
  }
}
"
`;
